de-DE.json

```diff
@@ -23,13 +23,13 @@
   "Collateral not found!": "Sicherheitskaution nicht gefunden!",
   "Complete": "Abgeschloßen",
   "Date & Time": "Datum & Zeit",
-  "Deposit ADA": "ADA Hinterlegen",
+  "Deposit ADA": "ADA hinterlegen",
   "Dive into the Top Liquidity Pools! Swap, Provide Liquidity and Yield Farm in selected Farm.": "Tauche ein in die top Liquiditäts Pools! Tausche, stelle Liquidität bereit und erhalte Erträge von ausgewählten Farmen.",
   "Dive into the Top Liquidity Pools! Swap, Provide Liquidity and Yield Farm in selected Farms.": "Tauche ein in die top Liquiditäts Pools! Tausche, stelle Liquidität bereit und erhalte Erträge von ausgewählten Farmen.",
   "Fee paid for the service of off-chain Laminar batcher to process transactions.": "Gebühr für den off-Chain Laminar Batcher um Transaktionen zu prozessieren.",
   "LP fee": "LP Gebühr",
   "Market Cap": "Marktkapitalisierung",
-  "No asset found.": "Keine Suchergebnisse.",
+  "No asset found.": "Kein Asset gefunden",
```

Comment:
Wird der String immer für eine Suche verwendet? Dann werde ich die Änderungen reverten.

Mein Gedanke ist, das der String ebenfalls für eine Fehlermeldung verwendet werden kann falls der Server Response fehlschlägt und Assets in einer Liste generell nicht angezeigt werden können.

Reply:
I think it's better to ask in #deutsch channel in Minswap as I don't think many members will look at Github PR.

Reply:
I asked Attila a german speaking Community Builder. Waiting for its response.

Reply:
> ja, ich glaube Asset passt - anscheinend wird es nur fuer assets verwendet

That means it's good.